packages/normalizr/src/schemas/Values.js

```diff
@@ -46,4 +46,8 @@ export default class ValuesSchema extends PolymorphicSchema {
       deleted,
     ];
   }
+
+  infer(args, indexes, recurse) {
+    return undefined;
```

Comment:
if infer isn’t required, then why define it when it returns undefined?

Reply:
it is required for all class cases. every algorithm has special code paths for the non class case but no other type discrimination otherwise.